content/post/2025-02-15-llm-codegen-and-you.md

```
@@ -0,0 +1,307 @@
+- run tests
+- rinse repeat ✩₊˚.⋆☾⋆⁺₊✧
+
+This is a pretty good way to incrementally improve a code base. It has been super helpful to do small amounts of work in a big code base. I have found that I can do any sized tasks with this method.
+
+### Prompt magic
+
+These quick hacks work super well to dig into places where we can make a project more robust. It is super quick, and effective.
+
+Here are some of my prompts that I use to dig into established code bases:
+
+#### Code review:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)